src/ripple/app/paths/RippleState.h

```diff
@@ -153,27 +151,27 @@ class RippleState
     getJson(int);
 
 private:
-    std::shared_ptr<SLE const> sle_;
-
-    bool mViewLowest;
+    uint256 key_;
 
-    std::uint32_t mFlags;
-
-    STAmount const& mLowLimit;
-    STAmount const& mHighLimit;
+    STAmount const mLowLimit;
+    STAmount const mHighLimit;
 
-    AccountID const& mLowID;
-    AccountID const& mHighID;
+    AccountID const mLowID;
+    AccountID const mHighID;
 
     Rate lowQualityIn_;
     Rate lowQualityOut_;
     Rate highQualityIn_;
     Rate highQualityOut_;
 
     STAmount mBalance;
+
```

Comment:
Note: investigate whether re-ordering all these variables can result in more effective packing for the structure.

Reply:
That's the reason I moved the `flags` and `viewLowest` members to the end. Right now there should be no unneeded padding, but we could move the `mbalance` before the `quality` since its larger.

Reply:
Fixed in `7643b21204 Remove lowID and highID from RippleState:`